src/js/common.js

```diff
@@ -153,12 +162,43 @@ module.exports.loadLatestAssets = (variant, openjdkImp, releaseType, release, ty
   queryAPI(release, url, openjdkImp, type, errorHandler, handleResponse);
 }
 
+module.exports.getChecksum = url => checksums[url]
+
+module.exports.loadChecksum = (url, message) => {
+  return new Promise((resolve, reject) => {
+    loadUrl(url, response => {
+      try {
+        const reader = new FileReader();
+        // This fires after the blob has been read/loaded.
+        reader.addEventListener('loadend', e => {
+          // todo@userquin: fix this -> https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement
+          console.info(e.srcElement);
+          console.info(e.target);
+          const text = e.srcElement.result;
+          const checksum = text.split(' ')[0].trim();
+          console.info(checksum);
+          checksums[url] = checksum;
+          resolve(message.replace('FILEHASH',checksum));
+        });
+        reader.readAsText(response.blob());
+      } catch (e) {
+        reject(e);
```

Comment:
What happens to the user experience on a rejection?

Reply:
in taht case the div container will not be shown, initially is hidden

Reply:
just see the staging server, there are errors downloading all sha256.txt files, and the div container is hidden, the page is the same as the original